File: runtime/go/units/rt/link_error.go
// A failed import resolution at instantiation time (missing import, or one of
// the wrong Go type), kept distinct from a trap.
type rtLinkError struct{ msg string }

func (e *rtLinkError) Error() string { return e.msg }

func (rt) link_error(msg string) {
    panic(&rtLinkError{msg})
}

File: runtime/java/units/rt/link_error.java
// A failed import resolution at instantiation time (missing import, or one of the wrong kind), kept distinct from a trap.
// `link_error` is void and throws.
static final class LinkError extends RuntimeException {
    LinkError(String msg) {
        super(msg);
    }
}

static void link_error(String msg) {
    throw new LinkError(msg);
}
